packages/netlify-cms-core/src/components/Editor/EditorControlPane/EditorControlPane.js

```diff
@@ -11,7 +18,24 @@ const ControlPaneContainer = styled.div`
   font-size: 16px;
 `;
 
+const getFieldValue = ({ field, entry, isTranslatable, locale }) => {
```

Comment:
For nested field:
```
- label: 'Object'
  name: 'object'
  widget: 'object'
  fields:
      - { label: 'String', name: 'string', widget: 'string', i18n: true }
      - { label: 'Boolean', name: 'boolean', widget: 'boolean', i18n: duplicate }
```
Not sure how it is possible to pull `string` value from `['data']` and the boolean value from `['i18n', 'locale', 'data']`. Besides the above sample object widget doesn't show up in the editor.